### DISCUSSION TITLE
Display default query template in Grafana CW plugin

### DISCUSSION THREAD

**Discussion by ainapudi** [grafana/grafana#48446] [Data Sources]
Requesting to display a default query template similar to CW log insights interface

**Comment by taleena** [comment]
Hi @ainapudi Would you be able to provide some more details/screenshots so that we can be clear on what you are requesting?

**Comment by ainapudi** [comment]
Hi @taleena The users want to see default query like in the image attached
![image (2)](https://user-images.githubusercontent.com/97984672/166080027-997534c3-b30a-436e-bdfc-91d6e39a229a.png)
.

**Comment by taleena** [comment]
Won't fix at this time, there is a wizard available to supply similar to what is being asked for here.
